src/components/TableOfContents/index.tsx

```diff
@@ -80,33 +83,25 @@ const TableOfContents = ({
       overflowY="auto"
       {...rest}
     >
-      <List {...outerListProps}>
-        {!hideEditButton && editPath && (
-          <ListItem mb={2}>
-            <ButtonLink
-              leftIcon={<Icon as={FaGithub} />}
-              href={editPath}
-              variant="outline"
-            >
-              {t("edit-page")}
-            </ButtonLink>
-          </ListItem>
-        )}
-        <ListItem>
-          <Box mb={2} textTransform="uppercase">
-            {t("on-this-page")}
-          </Box>
-          <List m={0}>
-            <ItemsList
-              items={items}
-              depth={0}
-              maxDepth={maxDepth ? maxDepth : 1}
-              activeHash={activeHash}
-            />
-          </List>
-        </ListItem>
+      {!hideEditButton && editPath && (
+        <ButtonLink
+          leftIcon={<Icon as={FaGithub} />}
+          href={editPath}
+          variant="outline"
+        >
+          {t("edit-page")}
+        </ButtonLink>
+      )}
+      <Box textTransform="uppercase">{t("on-this-page")}</Box>
```

Comment:
Figma:
<img width="850" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/16356e5d-49ae-45ea-9603-96136f4a64cf">

Preview:
<img width="556" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/55db96eb-0a79-4e42-ab90-82efdb45711c">
<img width="323" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/79c37aa7-25a7-44ce-adba-5b4bb571ddfa">

```suggestion
      <Box textTransform="uppercase" color="body.medium">
        {t("on-this-page")}
      </Box>
```